JSON converter work

jouva opened the pull request:
This will add the pontojson script as well as format the documentation for the repo

jouva committed:
First version of .pon to .json conversion utility

jouva committed:
Clean up READNE for github

CodesignDev commented on pontojson.py:
```diff
@@ -0,0 +1,178 @@
+    "Dungeon": "symbol-dungeon.png",
+    "2014": "symbol-2014.png",
+    "2015": "symbol-2015.png",
+    "Workshop": "symbol-workshop.png",
```
Is this needed, as there hasn't been a Workshop symbol in any of the current sets, or is this for future stuff?

marisathewitch replied:
It's all current sets.
and i thibk all picture now avaliable

jouva replied:
No there's a bunch that were in the expansion icon list but had no file. Namely:
- Adventure Pack
- Custom Cards
- Dungeon Delvers
- 2014 Con Exclusives
- 2015 Con Exclusives
- Hearthswarming
- Gracious Givers
- Weeaboo Paradaisu

This WAS however missing the Panels of Ponies expansion icon reference

jouva committed:
Clean up expansion icon list

- Removed all references to icons not in expansion icon folder
- Added Panels of Ponies (Comics) expansion icon

jouva commented on pontojson.py:
```diff
@@ -0,0 +1,151 @@
+"""
+__author__ = "Jouva Moufette"
+__copyright__ = "Copyright 2016, Children of Kefentse"
+__credits__ = ["", ""]
```
I don't know why this is still here

jouva commented on pontojson.py:
```diff
@@ -0,0 +1,151 @@
+__license__ = "CC-BY-SA"
+__version__ = "1.0.0"
+__maintainer__ = "Jouva Moufette"
+__email__ = "jouva@moufette.com"
```
I should change this now

n-a-larson commented on README.md:
```diff
@@ -33,29 +33,29 @@ The script will open the filepath to cards.pon (or whatever you name it), and wi
+* `{3swap}` = "You may swap up to 3 Pony cards on the grid."
+* `{draw}` = "You may draw a card from the Ship or Pony deck."
+* `{goal}` = "You may discard a Goal and draw a new one to replace it."
+* `{search}` = "You may search the Ship or Pony discard pile for a card of your choice and play it."
```
These are not the latest wordings

CodesignDev commented:
:+1:

n-a-larson commented on README.md:
```diff
@@ -33,29 +33,30 @@ The script will open the filepath to cards.pon (or whatever you name it), and wi
+* `{draw}` = "You may draw 1 card from the Ship or Pony deck."
+* `{goal}` = "You may discard 1 active Goal and draw 1 new Goal to replace it."
+* `{search}` = "You may search the Ship or Pony discard pile for 1 card of your choice and put it into your hand. If it's still in your hand at the end of your turn, discard it."
+* `{copy}` = "You may copy the power of any Pony card currently on the shipping grid, except for Changelings."
```
Replace "Changelings" with "Ponies with the Changeling keyword"

jouva replied:
Can you change this in the game module as well?

n-a-larson replied:
do you mean the CardMachine?

n-a-larson commented on README.md:
```diff
@@ -33,29 +33,30 @@ The script will open the filepath to cards.pon (or whatever you name it), and wi
+* `{love poison}` = "Instead of playing this Ship with a Pony card from your hand, or connecting two Pony cards already on the grid, you may take a Pony card from the shipping grid and reattach it elsewhere with this Ship. That card's power activates."
+* `{keyword change}` = "When you attach this card to the grid, you may choose one Pony card attached to this Ship. Until the end of your turn, that Pony card gains one keyword of your choice, except for Pony names."
+* `{gender change}` = "When you attach this card to the grid, you may choose one Pony card attached to this Ship. Until the end of your turn, that Pony card becomes the opposite gender."
+* `{race change}` = "When you attach this card to the grid, you may choose one Pony card attached to this Ship. Until the end of your turn, that Pony card becomes the race of your choice. This cannot affect Changelings."
```
Replace "Changelings" with "Ponies with the Changeling keyword"

jouva replied:
Can you change this in the game module as well?

n-a-larson commented:
:+1: I verified the rules-wordings, and I reviewed the python as well as I could.
